translations/es/LC_MESSAGES/messages.po

```diff
@@ -223,7 +228,7 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "Building the future web, together."
-msgstr "Armando el futuro de la web, juntos."
+msgstr "Juntos, Armando el futuro de la web."
```

Comment:
I can not edit this file here @bpaduch, or, maybe you can merge this file as it is, and then I'll edit. :+1: